inference-engine/src/transformations/src/transformations/common_optimizations/eliminate_unsqueeze_gather.cpp

```diff
@@ -5,6 +5,7 @@
 #include "transformations/common_optimizations/eliminate_unsqueeze_gather.hpp"
 
 #include <ngraph/opsets/opset6.hpp>
+#include <ngraph/opsets/opset7.hpp>
```

Comment:
We try to use single opset inside transformation if it doesn't convert operation from opsetX to opsetY. So if we start using opset7 then we need to use it for other ops. This change can be done in separate PR.

Reply:
changed to `GatherBase`